Add `GB Cup 2024 IRL` wiki articles (#12371)
ppy/osu-wiki: wiki/Tournaments/GBC/GBC_2024_IRL/en.md

---
tags:
  - GBC
  - GBC2024
  - GBC 2024
  - GBC 2024 IRL
  - GBC2024IRL
---

> # # GB Cup 2024 In Real Life

Reviewer: ```suggestion
# GB Cup 2024 In Real Life
```

![GB杯2024线下赛图片](img/gbc2024irl-banner.png)

> **GBC 2024 IRL** (***GB Cup 2024 In Real Life***) is hosted by [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) and organized by GB. It is an osu!mania 4k event that was held offline at the Kululu Music Game Carnival.

Author: according to my knowledge there is no link of Kululu Music Game Carnival, only some videos I guess

| --: | :-: | :-: | :-- | :-- |
| **[GB]GanyuAngel** | **4** | 0 | Shenzouz | [#1](https://osu.ppy.sh/community/matches/114831199) |
| [Crz]Riko | 0 | **4** | **Nyagato Yuki** | [#1](https://osu.ppy.sh/community/matches/114832202) |
| **Mantozi** | **4** | 1 | [GB]QAQlingjiu | [#1](https://osu.ppy.sh/community/matches/114832361) |
| **LazyGhost14** | **4** | 1 | [Crz]raber | [#1](https://osu.ppy.sh/community/matches/114831955) |
| **[GB]Mafufu** | **4** | 0 | vanposen | [#1](https://osu.ppy.sh/community/matches/114831457) |
| **DawnX** | **4** | 0 | [ Classic ] | [#1](https://osu.ppy.sh/community/matches/114831736) |
| **[Crz]Xinyi2016** | **4** | 1 | [GB]Burger King | [#1](https://osu.ppy.sh/community/matches/114830450) |
| **nyasunfanboy** | **4** | 1 | [Crz]Nickname | [#1](https://osu.ppy.sh/community/matches/114830678) |

### Regular Stage

Due to the large number of players and matches, the match results will be displayed in the **[information spreadsheet](https://docs.qq.com/sheet/DTWpxUkN6SGtrS29m?tab=j9vlga)**.

## Qualifier instructions

1. There are 8 maps in the qualifier mappool.
2. The ranking of qualifier is determined by sum of all the single maps ranking.

> 3. The top 16 players in the qualifier will enter the main competition. They will be divided into 4 tiers according to the player ranking and randomly assigned into 4 quarter zones, while every quarter will forcely include 1 player from each tier. The first round of the game is fixed as tier 1 vs tier 4, and tier 2 vs tier 3.

Reviewer: ```suggestion
3. The top 16 players in the qualifier will enter the main competition. They will be divided into 4 tiers according to the player ranking and randomly assigned into 4 quarter zones, where every quarter will forcibly include 1 player from each tier. The first round of the game is fixed as tier 1 vs. tier 4, and tier 2 vs. tier 3.
```

> 4. After the qualifier round, if any qualified player withdraws, his ranking will be replaced by the next player, and so on.

Reviewer: ```suggestion
4. After the qualifier round, if any qualified player withdraws, their ranking will be replaced by the next player, and so on.
```

## Tournament rules

1. There is no warm-up phase for the match.

> 2. The main competition is a single-elimination knockout competition with a total of 16 players. The round of 16 is a best-of-seven game, the quarterfinals and semifinals are a best-of-nine game, and the final is a best-of-eleven game.

Reviewer: ```suggestion
2. The main competition is a single-elimination knockout competition with a total of 16 players. The Round of 16 is a best-of-7 game, the Quarterfinals and Semifinals are a best-of-9 game, and the Final is a best-of-11 game.
```

> 3. There are three maps for each slot in the mappool. When selecting a map, you can only select the position. The specific map to be used will be determined by random roll on the spot.

Reviewer: ```suggestion
3. There are three maps for each slot in the mappool. When selecting a map, you can only select the position. The specific map to be used will be determined by random `!roll` on the spot.
```

> 4. During the protect phase, both players choose a slot and assign it to a specific map. The map of this slot will not be randomized in subsequent selections, if this slot got piced the map will directly be selected. Also this slot cannot be banned.

Reviewer: ```suggestion
4. During the protect phase, both players choose a slot and assign it to a specific map. The map of this slot will not be randomised in subsequent selections. If this slot gets picked, the map will directly be selected. Additionally, this slot cannot be banned.
```

5. During the ban phase, both players will ban all the maps of one slot.

> 6. When the two sides are tied and the last point is decided, the last map will use Tie-breaker. At this time, both sides will ban one map at the same time through the referee. If different maps are banned, the remaining one will be the map; if the same map is banned, one map will be randomly selected from the remaining two maps.

Reviewer: ```suggestion
6. When the two sides are tied and the last point is decided, the last map will be a tiebreaker. At this time, both sides will ban one map at the same time through the referee. If different maps are banned, the remaining one will be the map played; if the same map is banned, one map will be randomly selected from the remaining two maps.
```

> 7. Each side will have 2 minutes to choose a map. If the timeout is reached, the referee will randomly select an unselected map from the mappool. After the map is selected, both players have 2 minutes to get ready.

Reviewer: written out elsewhere in this article

```suggestion
7. Each side will have two minutes to choose a map. If the timeout is reached, the referee will randomly select an unselected map from the mappool. After the map is selected, both players have two minutes to get ready.
```

8. Possible mod choices for the FreeMod bracket are NoFail, Hidden, Fade In, Flashlight, and Mirror.

> 9. There are no restrictions on whether both players can choose or not choose a certain map, except that the selected map must be in the game map pool, and you cannot choose a map that has been selected in the game, and you cannot choose Tiebreaker.

Reviewer: ```suggestion
9. There are no restrictions on whether both players can choose or not choose a certain map, except that the selected map must be in the game mappool, and you cannot choose a map that has been selected in the game, and you cannot choose the tiebreaker.
```

10. In the case of a tie, the beatmap will be rematched, but not more than twice.

> 11. Players are allowed to rematch if any player disconnects early on during a beatmap. Send a message as soon as poosible in the match room chat to notify the referee to restart the match.

Reviewer: ```suggestion
11. Players are allowed to rematch if any player disconnects early on during a beatmap. Send a message as soon as possible in the match room chat to notify the referee to restart the match.
```

> 12. If necessary, the referee has the right to decide the implementation of the rules at his own discretion and has the final say on events that occur during the game.

Reviewer: ```suggestion
12. If necessary, the referee has the right to decide the implementation of the rules at their own discretion and has the final say on events that occur during the game.
```

---
tags:
  - GBC
  - GBC2024
  - GBC 2024
  - GBC 2024 IRL
  - GBC2024IRL
---

# GB Cup 2024 In Real Life

> ![GB杯2024线下赛图片](img/gbc2024irl-banner.png)

Reviewer: ```suggestion
![GB Cup 2024 IRL banner](img/gbc2024irl-banner.png)
```

> **GB Cup 2024 In Real Life** (***GBC 2024 IRL***) is a LAN osu!mania 4-key solo tournament hosted by [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) and organized by GB. It was the sixth instalment of the GB Cup series and the first instalment of the GB Cup IRL. This LAN tournament was held offline at the Kululu Music Game Carnival in Shanghai, China.

Reviewer: ```suggestion
**GB Cup 2024 In Real Life** (***GBC 2024 IRL***) is a osu!mania 4-key solo LAN tournament hosted by [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) and organised by Team GB. It was the sixth instalment of the GB Cup series and the first instalment of the GB Cup IRL. This LAN tournament was held offline at the Kululu Music Game Carnival in Shanghai, China.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2024-06-03/2024-06-30 |
| Qualifier stage | 2024-07-13/2024-07-14 |
| Offline tournament stage | 2024-07-27/2024-07-28 |

## Prizes

| Placing | Prizes |
| :-: | :-- |
| ![Golden Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, KanonBot profile badge, championship trophy, GBC pillow, GBC medal |
| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | KanonBot profile badge, 2nd place trophy, GBC pillow, GBC medal |
| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | KanonBot profile badge, 3rd place trophy, GBC pillow, GBC medal |
| *Others* | GBC pillow, GBC medal |

> ![Trophy Image](img/gbc2024irl-cups.jpg "GBC 2024 IRL trophies")
> 
> ![Badge Image](img/gbc2024irl-winner.png)

Reviewer: ```suggestion
![Trophy image](img/gbc2024irl-cups.jpg "GBC 2024 IRL trophies")

![Badge image](img/gbc2024irl-winner.png)
```

> ![Badge image](img/gbc2024irl-winner.png)

Reviewer: mind updating the translation for this part too

```suggestion
![Badge image](img/gbc2024irl-winner.png "GBC 2024 IRL winner badge")
```
